app/server/skychat/plugins/entertainment/StatsPlugin.ts

```diff
@@ -0,0 +1,64 @@
+import {Connection} from "../../Connection";
+import {Plugin} from "../../Plugin";
+import {UserController} from "../../UserController";
+import {User} from "../../User";
+import {Session} from "../../Session";
+
+export class StatsPlugin extends Plugin {
+
+    static readonly AVERAGE_BOOK_READ_TIME: number    = 60 * 5; 
```

Comment:
To be consistent with the rest of the code, we do not align variable declarations/definitions
Please could you switch to 
`static readonly AVERAGE_BOOK_READ_TIME: number = 60 * 5;`